js/features/supportEvents.js

```diff
@@ -12,6 +12,7 @@ function registerListeners(component, listeners) {
     listeners.forEach(name => {
         // Register a global listener...
         let handler = (e) => {
+           console.log('this one')
```

Comment:
@calebporzio I think you merged a console.log here :)

Reply:
Good catch! I've removed it https://github.com/livewire/livewire/pull/6811